de.tu_bs.cs.isf.e4cf.parts.project_explorer/ui/view/ProjectExplorerView.fxml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.ContextMenu?>
+<?import javafx.scene.control.MenuItem?>
+<?import javafx.scene.control.TreeView?>
+<?import javafx.scene.layout.BorderPane?>
+
+<BorderPane maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="400.0" prefWidth="600.0" xmlns="http://javafx.com/javafx/8.0.171" xmlns:fx="http://javafx.com/fxml/1" fx:controller="de.tu_bs.cs.isf.e4cf.parts.project_explorer.view.ProjectExplorerView">
```

Comment:
Wird wohl korrekt sein, aber was bedeutet hier `-Infinity`? Ich habe leider in der Doku nichts gefunden. Hätten da nicht die default values ausgereicht?

Reply:
Vielleicht könnt ihr mir ja ein Link oder so schicken :sweat_smile:

Reply:
Das ist damit das mit der größe des Parents wächst.